packages/transport/src/modules/relationships/RelationshipsController.ts

```diff
@@ -154,12 +154,22 @@ export class RelationshipsController extends TransportController {
 
         const { requestCipher, requestContent } = await this.prepareRequest(secretId, template, parameters.content);
 
-        const backboneResponse = (
-            await this.client.createRelationship({
-                content: requestCipher.toBase64(),
-                relationshipTemplateId: template.id.toString()
-            })
-        ).value;
+        let backboneResponse: BackboneGetRelationshipsResponse;
+        try {
+            backboneResponse = (
+                await this.client.createRelationship({
+                    content: requestCipher.toBase64(),
+                    relationshipTemplateId: template.id.toString()
+                })
+            ).value;
+        } catch (error: any) {
+            if (error.code === "error.platform.validation.relationship.peerIsToBeDeleted") {
+                throw new Error(
+                    "The Identity who created the RelationshipTemplate is currently in the process of deleting itself. Thus, it is not possible to establish a Relationship to it."
+                );
```

Comment:
shouldn't this be a CoreError with a proper error code?

Reply:
I thought we use CoreErrors with ErrorCodes only in case the user did something wrong.

Reply:
The user tried to open a relationship to an identity that is deleting itself. The user cannot really know this, but it is the users fault.

If we don't give out a CoreError, why do we need this PR at all?